docs/contents.md

```diff
@@ -61,17 +61,16 @@ The **OT - Vulnerability Management** solution pack contains the following resou
 | 02 - Use Case - ICS Advisory and CVEs |
 | :------------------------------------ |
 
-| Playbook Name                                        | Description                                                                                |
-| :--------------------------------------------------- | :----------------------------------------------------------------------------------------- |
-| Ingest CISA Advisories<sup>New<sup>                  | Playbook ingests CISA ICS Advisory and create record for same.                             |
-| Ingest Latest CISA ICS Advisory<sup>New<sup>         | Playbook ingests latest CISA ICS Advisory and create record for same.                      |
-| Ingest Latest CISA ICS Medical Advisory<sup>New<sup> | Playbook ingests latest CISA ICS Medical Advisory and create record for same.              |
-| On ICS Advisory Creation                             | Playbook executes post ICS Advisory Creation.                                              |
-| Correlate CVEs and Assets                            | Playbook correlate CVEs and Assets by vendor & product with Advisory.                      |
-| On ICS Advisory Update                               | Playbook executes when new CVE is added to ICS Advisory.                                   |
-| CVE Changes To KEV                                   | Playbook mark correlated ICS Advisory to KEV as CVE changes to KEV.                        |
-| Get Latest Report Of CVEs                            | Playbook executes on ICS Advisory and KEV Alert to fetch latest report of correlated CVEs. |
-| Get And Update CVE                                   | Playbook fetches latest report of CVE from NIST-NVD and update CVE record.                 |
+| Playbook Name                                 | Description                                                                                |
+| :-------------------------------------------- | :----------------------------------------------------------------------------------------- |
+| Ingest CISA ICS Advisory<sup>New<sup>         | Playbook ingests latest CISA ICS Advisory and create record for same.                      |
+| Ingest CISA ICS Medical Advisory<sup>New<sup> | Playbook ingests latest CISA ICS Medical Advisory and create record for same.              |
+| On ICS Advisory Creation                      | Playbook executes post ICS Advisory Creation.                                              |

```

Comment:
Why? What is a purpose of this PB

Reply:
Playbook executes on advisory creation and correlate Asset and CVE